src/Dropdown/Examples.md

```diff
@@ -1,184 +1,200 @@
 ### Examples
 
 ```js
-initialState = {}
```

Comment:
looks like the majority of changes to this file are just due to prettier (so out of scope for the PR). There's always the option to "File: save without formatting" in vscode. Or, to prevent this happening again, we could setup a prettierrc for this folder I guess